contrib/kubernetes/static/controllers-pod.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: openshift-master-controller
```

Comment:
This pod is what the point of this PR is for - to be able to run the controllers process as a static pod under the kubelet, but not depend on a connection to etcd (like the upstream controller manager).